src/node/flows.ml

```diff
@@ -132,7 +131,7 @@ let rec request_protocol_snapshot tries =
 let () =
   Lwt.async_exception_hook :=
     fun exn ->
-      Printexc.to_string exn |> Format.eprintf "global_exception: %s\n%!";
+      Log.error (Printexc.to_string exn |> Printf.sprintf "global_exception: %s");
```

Comment:
Another issue with this API is that we always assume strings, here we pay for conversion `exn -> string` and 2x string interpolation.
To make it slightly better we could do something like
```ocaml
Log.error (Printf.sprintf "global_exception: %a" Fmt.exn exn);
```
But it's still as expensive just slightly nicer, and we pay for most of it even if we didn't want to log errors for some reason.